### PR TITLE
Persian language

### PR DESCRIPTION
**Pull request by CodeWithEmad**
This will add Persian (fa_IR) language to the Authn MFE.
Additionally, the translations for other languages have been updated, and a few typos have been corrected.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @CodeWithEmad!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

**Comment by mphilbrick211** [comment]
Hi @CodeWithEmad! Just flagging that there are a couple failing checks. Thanks!

**Comment by CodeWithEmad** [comment]
Hi @mphilbrick211 
Thank you for letting me know. The issue has been resolved.

**Comment by e0d** [comment]
@CodeWithEmad looks like there are some CI failures, can you have a look?

**Comment by CodeWithEmad** [comment]
consider it done @e0d!

File: package-lock.json [hunk 1]
```diff
@@ -22167,9 +22167,9 @@
       "integrity": "sha512-CC1bOL87PIWSBhDcTrdeLo6eGT7mCFtrg0uIJtqJUFyK+eJnzl8A1niH56uu7KMa5XFrtiV+AQuHO3n7DsHnLQ=="
     },
     "node_modules/word-wrap": {
-      "version": "1.2.3",
-      "resolved": "https://registry.npmjs.org/word-wrap/-/word-wrap-1.2.3.tgz",
-      "integrity": "sha512-Hz/mrNwitNRh/HUAtM/VT/5VH+ygD6DV7mYKZAtHOrbs8U7lvPS6xf7EJKMF0uW1KJCl0H701g3ZGus+muE5vQ==",
+      "version": "1.2.5",
+      "resolved": "https://registry.npmjs.org/word-wrap/-/word-wrap-1.2.5.tgz",
+      "integrity": "sha512-BN22B5eaMMI9UMtjrGd5g5eCYPpCPDUy0FJXbYsaT5zYxjFOckS53SQDE3pWkVoWpHXVb3BrYcEN4Twa55B5cA==",
```
Comment (zainab-amir) [inline package-lock.json:22172]: Is this change intentional?

Comment (CodeWithEmad) [reply package-lock.json:22172]: no, it wasn't. I can remove the commit if it's not acceptable :)

Comment (zainab-amir) [reply package-lock.json:22172]: yes, please.

Comment (CodeWithEmad) [reply package-lock.json]: Done

**Review by zainab-amir — APPROVED** [review]
